### DISCUSSION TITLE
case insensitive regex path

### DISCUSSION THREAD

**Discussion by fedoranvar** [grafana/grafana-json-datasource#121] [Q&A]
Good day to you!

I've query-filter,. with variable `$document`:
`$.result[?(@.item.match('.*$document.*'))].item`

I want make it case-insensitive.

Tried 
`$.result[?(@.item.match('/.*$document.*/i'))].item`

No success.

Is it possible to do case-insensitive query?

thank you!

**Comment by marcusolsson** [comment]
Good question! I would check the [JSONPath Plus](https://www.npmjs.com/package/jsonpath-plus) for documentation on this. If they support it, it should be possible.

**Reply by fedoranvar** [reply]
@marcusolsson  Link you provided shows example, which uses `case-insensitive` flag
![image](https://user-images.githubusercontent.com/52028706/121365127-d18aca80-c951-11eb-8679-fe3979d25a7d.png)

I've tried to implement it in grafana - but no luck.

Could check this, please?

**Reply by marcusolsson** [reply]
Thank you! If JSONPath Plus claims it's possible, it might be a bug. Could you create an issue for this? Preferably with some sample JSON and the query you expect to work.
